lib/heroku/command/pg.rb

```diff
@@ -14,6 +14,16 @@ def set_commands(shorthand)
     end
   end
 
+  # pg:stats_reset
+  # calls the Postgres functions pg_stat_reset()
```

Comment:
@mandagill Try taking a first pass at the documentation. Then rope me in and we can review it together.